exonum/src/runtime/types.rs

```diff
@@ -275,6 +275,16 @@ impl Display for ArtifactId {
     }
 }
 
+impl Debug for ArtifactId {
```

Comment:
Are this and the following `Debug` implementations motivated by the `non_exhaustive` marker? IMO, it's OK to output in `Debug`; it deals with the internal type structure anyway.

Reply:
But it appears in the log. This is a reason why i’ve reimplement Debug.

Reply:
I think the problem here is that log entry uses debug representation (which is misuse). Log entries should use `Display`.